modules/ossm-configuring-distr-tracing-tempo.adoc

```diff
@@ -66,35 +140,61 @@ kind: Kiali
 spec:
   external_services:
     tracing:
-      query_timeout: 30
+      query_timeout: 30 #<1>
       enabled: true
       in_cluster_url: 'http://tempo-sample-query-frontend.tracing-system.svc.cluster.local:16685'
       url: '[Tempo query frontend Route url]'
-      use_grpc: true # <1>
+      use_grpc: true # <2>
 ----
-<1> If you are not using the default HTTP or gRPC port for Jaeger or Tempo, replace the `in_cluster_url:` port with your custom port.
+<1> The default `query_timeout` is 30. If you set it greater than 30, you need to update `.spec.server.write_timeout` in the Kiali CR and add the annotation `haproxy.router.openshift.io/timeout=50s` to the Kiali route. Both `.spec.server.write_timeout` and `haproxy.router.openshift.io/timeout=` must be greater than `query_timeout`.
```

Comment:
```suggestion
<1> The default `query_timeout` value is 30 seconds. If you set the value to greater than 30 seconds, you must update `.spec.server.write_timeout` in the Kiali CR and add the annotation `haproxy.router.openshift.io/timeout=50s` to the Kiali route. Both `.spec.server.write_timeout` and `haproxy.router.openshift.io/timeout=` must be greater than `query_timeout`.
```
Is this seconds? Can I use a fractional value like 35.5? Probably a good idea to define the valid values and syntax for them.